src/lib.rs

```diff
@@ -243,7 +243,7 @@ impl<T: Texture> Cache<T> {
 /// ['Text Rendering Hates You'](https://gankra.github.io/blah/text-hates-you) for more information
 /// on why text is complicated.
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
-pub struct Glyph(pub u32);
+pub struct Glyph(pub u16);
```

Comment:
This is a breaking change. Is it necessary?

Reply:
I guess not, though both the backends have switched to using u16 for glyphs.